### ISSUE TITLE
Not working with Windows, "Object has no method 'isAbsolute'"

### ISSUE THREAD

**Issue by mkody** [1000ch/node-github-markdown#9]
I'm using node-github-markdown on Windows (XP) at work, and with the Sublime Text 3 Build system.
Everything was working fine on v0.4.2 (3e71971f6587fa548f1f91960009ceb914fddc80), but with the latests version, I'm getting a `TypeError: Object #<Object> has no method 'isAbsolute'` error.
In this example, my files are inside a second partition, H:, at `H:\Documents\sql`.
I've tried to use it directly from CLI with just a `ghmd "prez.md"` but I got the exact same output:
```
C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\index.js:31
  if (!path.isAbsolute(this.template)) {
            ^
TypeError: Object #<Object> has no method 'isAbsolute'
    at new module.exports (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\index.js:31:13)
    at C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\cmd.js:57:16
    at C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:159:20
    at C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:227:13
    at _arrayEach (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:78:13)
    at _each (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:69:13)
    at Object.async.forEachOf.async.eachOf (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:226:9)
    at Object.async.forEach.async.each (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\async\lib\async.js:203:22)
    at C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\cmd.js:36:9
    at module.exports (C:\Documents and Settings\André Fernandes\Application Data\npm\node_modules\github-markdown\node_modules\ls-path\index.js:40:5)
[Finished in 2.2s with exit code 8]
...
```

**node** v0.10.26
**npm** 1.4.3

**Comment by 1000ch** [comment]
**v0.10.x** `path` module does not have `isAbsolute()` method.
I fix it with `path-is-absolute` ponyfill. And I recommend you to use **v0.12.x**.

**Comment by 1000ch** [comment]
This is fixed on `v1.1.4`. Would you update?
https://www.npmjs.com/package/github-markdown
